website/test/html/ChangePluginsSample.html

```diff
@@ -0,0 +1,110 @@
+<!DOCTYPE html>
+    <head>
+        <link rel="stylesheet" type="text/css" href="../css/toggleButtons.css">
+        <link rel="stylesheet" type="text/css" href="../../styles/main.css">
+    </head>
+    <body>       
+             <!--advanced all plugins-->
+             <div id="allPlugins-panel" class="bg-black border-gray" style="width: 30vw;color:white;height: 100vh;overflow-y: scroll;">
+              <div class="card text-white bg-black mb-3" style="width: 100%;">                    
+                  <div class="card-header card-custom-header" style="text-align: center;padding-left: 0px;">
+                      <button type="button" class="btn btn-info" id="changePluginsCloseBtn">X</button>
+                      <div>
+                         Change Plugins
+                      </div> 
+                  </div>
+                  <div class="card-body" >
+                    <p class="card-text warning-messages">
+                        <i class="fas fa-exclamation-triangle"></i>
+                        <span>Please proceed with caution, any changes to these plugins will change vehicle behavior</span>                       
```

Comment:
Please update to : 
"Please proceed with caution. Any changes to these plugins will change vehicle behavior."

Reply:
https://github.com/usdot-fhwa-stol/carma-web-ui/commit/3b75b3d4bd09d69cb18eac69579c8e5015d802cb